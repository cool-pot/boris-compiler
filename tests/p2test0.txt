*** Hello, world!
*** Except we have no way to print strings.
print 131313;