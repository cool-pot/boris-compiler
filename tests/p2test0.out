> Hello from parserTester for boris compiler!
Start parse tree visualization:

[ROOT_INPUT]
    [SDD_LIST]
        [STATEMENT_AS_SDD]
            [PRINT_STATEMENT]
                [PLACEHOLDER-274]print
                [SINGLE_INT_AS_EXPR]
                    [INT]131313
                [PLACEHOLDER-290];
        [EMPTY_NODE]
